botcity/maestro/impl/interface.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, server: Optional[str] = None, login: Optional[str] = None, ke
         self._access_token = None
         self._task_id = 0
         self._impl: BotMaestroSDKInterface = None  # type: ignore
-        self._version = None
+        self._version = ""
```

Comment:
Why was this chance necessary?

Reply:
It was pointing out a type error by Mypy, but after updating the .ini it ended up not being necessary anymore.
I'm going back to the previous way.